src/soft_assay_rules/rule_generator.py

```diff
@@ -30,7 +30,7 @@
      "rule_description": "Preamble rule identifying non-DCWG"
      },
     {"type": "note",
-     "match": "not_dcwg and assay_type == null and data_types != null",
+     "match": "not_dcwg and ((assay_type == null and data_types != null) or entity_type == 'Publication')",
```

Comment:
OK, I acknowledge that Publications seem to be consistently described as derived.  This breaks the model in other ways though, because they have a metadata.tsv file, unlike any other kind of derived dataset.  Fortuntely a rule to recognize publications can be based solely on the entity_type, so I think there is no practical issue.

Reply:
@jswelling - I think the model will be broken in the future regardless since EPICs are external derived datasets that will have metadata.tsv files associated with them.

That being said, I'd rather this rule correctly represent Publications. That is, if they are actually "primary dataset" then we should update the assay_type file to reflect that, and do away with this weird edge case.